### DISCUSSION TITLE
The routine load task consumes uneven Kafka data

### DISCUSSION THREAD

**Discussion by allen-gf** [apache/doris#29700] [A - General / Q&A]
doris版本：v2.0.2
kafka集群某topic分区有10个，发现routine load运行几个小时之后会自动挂掉报错为
ReasonOfStateChanged: ErrorReason{code=errCode = 104, msg='be 107892 abort task with reason: [INTERNAL_ERROR]fetch failed due to requested offset not available on the broker: Broker: Offset out of range

我理解这个报错是因为我限制kafaka 6h之前的数据会删除掉导致的，感觉导致这个问题的根本原因是不分分区消费不均匀延迟过大导致，为什么会有消费不均匀的问题，一直没查到

**Comment by yagagagaga** [comment]
You can view more information by exec `show routine load` and see what content of the `Progress` and `Lag` field.

**Comment by nospeak12345** [comment]
i have the same problem too.但是原因不清楚，确实有的分区消费的快，有的消费的慢，but my version is palo1.1.0

**Reply by allen-gf** [reply]
后来发现应该是消费线程调节太少的原因
